src/control/cmd/dmg/pool_test.go

```diff
@@ -857,18 +857,6 @@ func TestPoolCommands(t *testing.T) {
 			"",
 			errors.New("can't set perf_domain on existing pool."),
 		},
-		{
-			"Set pool rd_fac property is not allowed",
-			"pool set-prop 031bcaf8-f0f5-42ef-b3c5-ee048676dceb rd_fac:1",
-			"",
-			errors.New("can't set redundancy factor on existing pool."),
-		},
```

Comment:
Might be worth adding a test case showing it can now be modified, similar to others above.

Reply:
yes I think this is important to keep up test coverage and prevent regressions